setup.py

```diff
@@ -13,16 +13,16 @@
     description=DESCRIPTION,
     long_description=LONG_DESCRIPTION,
     packages=find_packages(),
+    include_package_data=True,  # Automatically include non-Python files
+    package_data={'': ['graphql/**/*.gql', 'abis/*.json']},
```

Comment:
```suggestion
    package_data={'': ['graphql/**/*.gql', 'abis/*.json', 'config/*.json']},
```

Reply:
```
>>> b = bal_addresses.pools_gauges.BalPoolsGauges('arbitrum', False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/gosuto/.local/share/virtualenvs/python-H43gZfHs/lib/python3.10/site-packages/bal_addresses/pools_gauges.py", line 22, in __init__
    self.core_pools = self.build_core_pools()
  File "/Users/gosuto/.local/share/virtualenvs/python-H43gZfHs/lib/python3.10/site-packages/bal_addresses/pools_gauges.py", line 174, in build_core_pools
    with open("config/core_pools_whitelist.json", "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'config/core_pools_whitelist.json'
```

Reply:
I don't think config should be included in the package.  It should always be pulled from main via an http request.    It may change between releases, and we want toe most recent config not the config from time at release.


really most of the config should probs be in multisig ops with more/different change control.